mods/page_revisions.js

```diff
@@ -76,6 +76,33 @@ PRS.prototype.getTableSchema = function () {
     };
 };
 
+/**
+ * Checks the revision info returned from the storage
+ * for restrictions, and if there are any, raises an error
+ *
+ * @param res Object the result as returned from storage
+ * @return Object the same object
+ * @throws rbUtil.httpError if access to the revision should be denied
+ */
+PRS.prototype._checkRevReturn = function(res) {
+    var item = res && res.body && Array.isArray(res.body.items) && res.body.items[0];
```

Comment:
I think the table storage API guarantees that a successful reply includes an items array, so we should be able to only check for its length.